### DISCUSSION TITLE
Adding Default Value to Content-Type JSON

### DISCUSSION THREAD

**Discussion by ghufr** [strapi/strapi#6974] [Legacy Topics]
- [ ] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**

**Please describe your feature request:**

sometimes it's good to have initial value in JSON field so we don't have to do null checking.

if JSON field is empty (null), this code will throw an error:

```js
// Increment jsonField.count value by one

await strapi.plugins["users-permissions"].services.user.edit({
  id
}, {
  $inc: {
    "jsonField.count": 1
  },
});
```
resulted:  `error MongoError: Cannot create field 'count' in element {jsonField: null} `
solution: set jsonField to be an empty object `jsonField = {}`

Thanks

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
